pkg/cosign/tuf/client.go

```diff
@@ -143,9 +154,9 @@ func getMetadataStatus(b []byte) (*MetadataStatus, error) {
 }
 
 func (t *TUF) getRootStatus() (*RootStatus, error) {
-	local := "embedded"
-	if !t.embedded {
-		local = rootCacheDir()
+	local := rootCacheDir()
+	if noCache() {
+		local = "in-memory"
```

Comment:
Oh, this is for logging purposes (local is a `string`): so that on initialize we print the location of the file system local store, or if in-memory, the best we can do is say the local is "in-memory"

Reply:
Oh yep, I mixed up this with the TUF client `local` var